hr_holidays_public/models/hr_public_holidays_line.py

```diff
@@ -0,0 +1,68 @@
+# ©  2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# ©  2014 initOS GmbH & Co. KG <http://www.initos.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import Warning as UserError
+
+
+class HrPublicHolidaysLine(models.Model):
+    _name = 'hr.holidays.public.line'
+    _description = 'Public Holidays Lines'
+    _order = "date, name desc"
+
+    name = fields.Char(
+        'Name',
+        required=True,
+    )
+    date = fields.Date(
+        'Date',
+        required=True
+    )
+    year_id = fields.Many2one(
+        'hr.holidays.public',
+        'Calendar Year',
+        required=True,
+    )
+    variable = fields.Boolean('Date may change')
```

Comment:
unclear the purpose of the field

Reply:
@elicoidal I don't remember why, the field is seems is not used, i think the purpose was that it should allow changing the date when you copy the holidays into a new year...